src/app/core/admin/feedback/admin-list-feedback.component.scss

```diff
@@ -0,0 +1,35 @@
+@import 'src/styles/colors';
+
+$dropdown-min-width: 100px;
+
+.dropdown-menu {
+	min-width: $dropdown-min-width;
+}
+
+.feedback-body {
+	width: 300px;
+	white-space: pre-wrap;
+}
+
+.feedback-status-button {
+	min-width: $dropdown-min-width;
+	border-radius: 15px;
+	border-width: 2px;
+	border-style: solid;
+}
+
+// TODO - do I need to use ux presets? I didn't find ones that looked good here
```

Comment:
@sshah-asymmetrik can / should these border-colors and background-colors be different opacities or some percentage darker / lighter of the base color? Would that help to keep consistency?

Reply:
Great idea! I used the `lighten` function and it works great!